SharpSploit/Execution/DynamicInvoke/Generic.cs

```diff
@@ -69,6 +72,79 @@ public static IntPtr LoadModuleFromDisk(string DLLPath)
             return hModule;
         }
 
+        /// <summary>
```

Comment:
We have a lot of logic here in this file that I don't feel is necessarily specific to `DynamicInvoke`. Specifically, all the Manual Mapping, Module Overloading, and Syscall functionality, I feel like could possible be split from the DynamicInvoke namespace?

I'm curious your thoughts on this @TheWover @FuzzySecurity. I know that after the Manual Mapping/Module Overloading/Syscall stub, a user is likely going to want to use DynamicInvoke on the result they get back. But I feel like the actual function of Manual Mapping/Module Overloading/Syscall is not actually related to DynamicInvoke.

I'm curious if you guys agree, and if so, how/where do you think this could be split out to?

Reply:
I would like to keep the syscall functionality within DynamicInvoke, but I would be okay with moving the manual mapping stuff. Maybe to SharpSploit.Execution.ManualMap?